### ISSUE TITLE
Uncaught TypeError: Cannot read property 'matchesParams' of undefined when hit enter in html file

### ISSUE THREAD

**Issue by onepill** [atom/atom#6899]
[Enter steps to reproduce below:]

![screenshot 2015-05-22 15 21 54](https://cloud.githubusercontent.com/assets/817262/7765922/4e032034-0096-11e5-82e7-fc65b62d75ac.png)
1. hit enter where the cursor shows in the above screenshot

**Atom Version**: 0.201.0
**System**: Mac OS X 10.10.4
**Thrown From**: Atom Core
### Stack Trace

Uncaught TypeError: Cannot read property 'matchesParams' of undefined

```
At /Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/marker-store.js:134

TypeError: Cannot read property 'matchesParams' of undefined
  at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/marker-store.js:134:21
  at Set.forEach (native)
  at MarkerStore.module.exports.MarkerStore.findMarkers (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/marker-store.js:130:17)
  at TextBuffer.module.exports.TextBuffer.findMarkers (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:620:31)
  at DisplayBuffer.module.exports.DisplayBuffer.findMarkers (/Applications/Atom.app/Contents/Resources/app.asar/src/display-buffer.js:1319:26)
  at DisplayBuffer.module.exports.DisplayBuffer.decorationsForScreenRowRange (/Applications/Atom.app/Contents/Resources/app.asar/src/display-buffer.js:1208:20)
  at TextEditor.module.exports.TextEditor.decorationsForScreenRowRange (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:1210:33)
  at TextEditorPresenter.module.exports.TextEditorPresenter.updateDecorations (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor-presenter.js:1500:26)
  at TextEditorPresenter.module.exports.TextEditorPresenter.getState (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor-presenter.js:122:14)
  at TextEditorComponent.module.exports.TextEditorComponent.updateSync (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor-component.js:182:38)
  at TextEditorComponent.module.exports.TextEditorComponent.becameVisible (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor-component.js:278:21)
  at TextEditorComponent.module.exports.TextEditorComponent.checkForVisibilityChange (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor-component.js:810:16)
  at TextEditorComponent.module.exports.TextEditorComponent.pollDOM (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor-component.js:797:17)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/text-editor-component.js:3:61
  at ViewRegistry.module.exports.ViewRegistry.performDocumentPoll (/Applications/Atom.app/Contents/Resources/app.asar/src/view-registry.js:202:11)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/view-registry.js:3:61

```
### Commands

```
  2x -0:34.4.0 pane:show-previous-item (atom-text-editor.editor.is-focused)
  3x -0:33.2.0 pane:show-next-item (atom-text-editor.editor.is-focused)
     -0:20.3.0 core:undo (atom-text-editor.editor.is-focused)
     -0:19.6.0 core:redo (atom-text-editor.editor.is-focused)
     -0:19.3.0 core:save (atom-text-editor.editor.is-focused)
     -0:17.8.0 editor:move-to-first-character-of-line (atom-text-editor.editor.is-focused)
  3x -0:17.5.0 atomic-emacs:previous-line (atom-text-editor.editor.is-focused)
     -0:17 atomic-emacs:set-mark (atom-text-editor.editor.is-focused)
  2x -0:16.8.0 atomic-emacs:next-line (atom-text-editor.editor.is-focused)
     -0:16.5.0 editor:move-to-end-of-line (atom-text-editor.editor.is-focused)
     -0:16 atomic-emacs:copy (atom-text-editor.editor.is-focused)
     -0:15.6.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:15.3.0 core:paste (atom-text-editor.editor.is-focused)
  2x -0:11 editor:newline (atom-text-editor.editor.is-focused)
     -0:03.4.0 atomic-emacs:next-line (atom-text-editor.editor.is-focused)
     -0:03.1.0 editor:move-to-first-character-of-line (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "seti-ui",
      "seti-syntax"
    ],
    "disabledPackages": [
      "emacs-mode"
    ],
    "projectHome": "~/projects",
    "ignoredNames": [
      ".git",
      ".hg",
      ".svn",
      ".DS_Store",
      "._*",
      "Thumbs.db",
      "bower_compponenets",
      "node_modules"
    ]
  },
  "editor": {
    "invisibles": {},
    "fontSize": 12,
    "lineHeight": 1.5
  }
}
```
### Installed Packages

``` coffee
# User
atomic-emacs, v0.5.1
seti-syntax, v0.4.0
seti-ui, v0.7.1

# Dev
No dev packages
```

**Comment by izuzak** [comment]
Thanks for the report, @onepill. Can you reliably reproduce this perhaps?

I wasn't able to reproduce this, but noticed that `pollDOM` call which we talked about recently, @kevinsawicki (I think we were looking at https://github.com/atom/atom/issues/6117? not sure). Maybe the same type of problem here? 

Also gonna /cc @nathansobo @maxbrunsfeld for more :eyes: on these types of problems

**Comment by nathansobo** [comment]
@maxbrunsfeld it seems like we're pulling a marker id out of the marker index that doesn't correspond to a marker object in the marker store. But how? My money is on a bug in the marker index deleting ids somehow. Maybe we should add that to the random spec?

**Comment by nathansobo** [comment]
Nevermind. Delete is in the random spec.

**Comment by nathansobo** [comment]
@izuzak Interestingly this doesn't seem directly related to entering the newline. The `pollDOM` is detecting an editor becoming visible and triggering a display update, in which the exception occurs because of the inconsistent state discussed above. The `MarkerStore` contains a `MarkerIndex` to optimize lookups. The index just stores integer ids, which map to objects representing markers in the marker store, and for some reason these are getting out of sync.

**Comment by nathansobo** [comment]
This could be some evil interaction with atomic-emacs. It's still a core bug. No package should be able to put the marker index in an inconsistent state. But it might be helpful in reproducing. Looks like the `atomic-emacs:set-mark` command was used recently before the issue, which probably creates a marker... or it could be a red herring.

**Comment by onepill** [comment]
@izuzak Hi, cant reproduce it after i closed and re-open that file.

**Comment by oggy** [comment]
I can reliably reproduce this by running `atomic-emacs:set-mark` (cmd-space), moving down a line, and then `editor:split-selections-into-lines` (cmd-shift-L).

Definitely seems like some interaction between AE & core, but I don't think we're doing anything outside the public API of Marker here. Will dig in more later.

**Comment by nathansobo** [comment]
I can confirm these reproduction steps. Thanks very much. Investigating this now.

**Comment by nathansobo** [comment]
Figured it out. When a destroyed marker is updated we are reinserting it into the index. Will push a fix later today when I'm done traveling.

**Comment by oggy** [comment]
Great -- thanks for the update @nathansobo !

**Comment by nathansobo** [comment]
:sparkles: Thanks a ton @oggy for the clear reproduction steps. You've saved us a lot of time.

**Comment by PointedEars** [comment]
**This problem appears to be back.**

1. Editing a LaTeX document
2. Error message, editor freezes (if I try to close the tab, second error message on the bottom)

**Atom**: 1.14.0 x64 (**no problems with 1.13.1**)
**Electron**: 1.3.13
**OS**: Debian GNU/Linux
**Thrown From**: Atom Core


### Stack Trace

Uncaught TypeError: Cannot read property 'matchesParams' of undefined

```
At /usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/marker-layer.js:191

TypeError: Cannot read property 'matchesParams' of undefined
    at /app.asar/node_modules/text-buffer/lib/marker-layer.js:191:22
    at Set.forEach (native)
    at MarkerLayer.module.exports.MarkerLayer.findMarkers (/app.asar/node_modules/text-buffer/lib/marker-layer.js:187:17)
    at DisplayMarkerLayer.module.exports.DisplayMarkerLayer.findMarkers (/app.asar/node_modules/text-buffer/lib/display-marker-layer.js:147:37)
    at DecorationManager.module.exports.DecorationManager.decorationsStateForScreenRowRange (/app.asar/src/decoration-manager.js:151:21)
    at TextEditor.module.exports.TextEditor.decorationsStateForScreenRowRange (/app.asar/src/text-editor.js:1698:37)
    at TextEditorPresenter.module.exports.TextEditorPresenter.fetchDecorations (/app.asar/src/text-editor-presenter.js:1381:44)
    at TextEditorPresenter.module.exports.TextEditorPresenter.getPreMeasurementState (/app.asar/src/text-editor-presenter.js:125:14)
    at TextEditorComponent.module.exports.TextEditorComponent.updateSyncPreMeasurement (/app.asar/src/text-editor-component.js:264:60)
    at TextEditorComponent.module.exports.TextEditorComponent.updateSync (/app.asar/src/text-editor-component.js:196:12)
    at TextEditorComponent.module.exports.TextEditorComponent.becameVisible (/app.asar/src/text-editor-component.js:303:21)
    at TextEditorComponent.module.exports.TextEditorComponent.checkForVisibilityChange (/app.asar/src/text-editor-component.js:994:16)
    at TextEditorComponent.module.exports.TextEditorComponent.pollDOM (/app.asar/src/text-editor-component.js:980:17)
    at /app.asar/src/text-editor-component.js:3:59
    at ViewRegistry.module.exports.ViewRegistry.performDocumentPoll (/app.asar/src/view-registry.js:263:9)
    at ViewRegistry.module.exports.ViewRegistry.performDocumentUpdate (/app.asar/src/view-registry.js:226:14)
    at /app.asar/src/view-registry.js:3:59
```

### Commands

```
     -1:05.5.0 core:save (input.hidden-input)
 17x -1:01.3.0 click (span.syntax--meta.syntax--function.syntax--environment.syntax--math.syntax--latex)
     -0:49.1.0 command-palette:toggle (input.hidden-input)
  3x -0:48.3.0 blur (input.hidden-input)
     -0:45.5.0 editor:consolidate-selections (input.hidden-input)
     -0:45.5.0 core:cancel (input.hidden-input)
 10x -0:45.4.0 blur (input.hidden-input)
     -0:41.6.0 click (div.title.icon.tex-icon.medium-blue)
  5x -0:40.8.0 blur (input.hidden-input)
     -0:40.6.0 click (div.title.icon.tex-icon.medium-blue)
  5x -0:40.6.0 blur (ul.list-inline.tab-bar.inset-panel)
     -0:31.7.0 click (a.btn-issue.btn.btn-error)
  2x -0:30.5.0 blur (a.btn-issue.btn.btn-error)
     -0:06.2.0 click (span.syntax--support.syntax--function.syntax--section.syntax--latex)
     -0:01.3.0 blur (a.btn-issue.btn.btn-error)
     -0:01.2.0 click (a.btn-copy-report.icon.icon-clippy)
```

### Non-Core Packages

```
atom-beautify 0.29.17 
atom-css-unit-converter 1.1.1 
atom-php-navigation 0.6.0 
atom-ternjs 0.17.0 
atom-x 0.4.0 
atomatigit 1.5.5 
autoclose-html 0.23.0 
autocomplete-awk 0.1.2 
autocomplete-bibtex 0.7.0 
autocomplete-clang 0.10.0 
autocomplete-html-entities 0.1.0 
autocomplete-paths 1.0.5 
autocomplete-php 0.3.7 
autocomplete-python undefined 
autocomplete-xml 0.9.4 
autoupdate-packages 1.3.1 
block-comment 0.4.3 
branch-status 0.8.3 
character-table 0.4.3 
clipboard-plus 0.5.1 
color-picker 2.2.5 
column-select 0.2.0 
compare-files 0.8.0 
css-snippets 1.1.0 
ctrl-last-tab 0.3.0 
docblockr 0.9.1 
emmet 2.4.3 
emmet-snippets-compatibility 1.0.6 
es-navigation 0.1.5 
es6-javascript 1.0.0 
ex-mode 0.13.1 
file-icons 2.0.15 
filesize 2.0.4 
fixmyjs 1.4.0 
fold-comments 0.6.0 
format-sql 0.3.0 
git-blame 0.4.11 
git-clone 0.2.0 
git-control 0.8.2 
git-diff-details 1.4.0 
git-difftool 0.2.7 
git-go 1.0.5 
git-grep 0.9.0 
git-hide 1.0.0 
git-history 3.2.0 
git-log 0.4.1 
git-plus 7.2.1 
git-status 0.3.4 
git-time-machine 1.5.4 
gitignore-snippets 0.2.5 
goto 1.8.3 
goto-definition 1.1.19 
highlight-selected undefined 
hyperclick 0.0.40 
hyperclick-php 0.2.1 
indent-toggle-on-paste 0.1.3 
iso8601-timestamp 0.1.1 
javascript-snippets 1.2.1 
js-hyperclick 1.10.1 
js-refactor undefined 
keyboard-localization 1.5.0 
language-apache 1.7.0 
language-diff 0.7.0 
language-gettext 0.6.1 
language-latex 1.0.0 
language-sql-mysql 0.3.1 
latex-autocomplete 1.0.0 
latex-completions 0.3.2 
latex-friend 0.0.11 
latex-hyperclick 0.2.0 
latexer 0.3.0 
latextools 0.8.5 
lcars-syntax 0.0.1 
less-autocompile 1.0.0 
less-than-slash 0.16.0 
line-count-status 0.0.3 
line-jumper 0.14.1 
linter 1.11.21 
linter-checkbashisms 0.1.5 
linter-chktex 1.3.0 
linter-csslint 1.3.3 
linter-htmlhint 1.3.1 
linter-jshint 3.0.2 
linter-less 2.6.0 
linter-mixed-indent 1.1.0 
linter-mypy 2.0.3 
linter-php 1.3.1 
linter-pylint 1.2.2 
linter-shellcheck 1.4.3 
linter-xmllint 1.4.0 
local-history 4.0.1 
man 0.8.2 
merge-conflicts 1.4.4 
method-separator 0.1.4 
minimap 4.26.8 
minimap-cursorline 0.2.0 
minimap-find-and-replace 4.5.1 
minimap-git-diff 4.3.1 
minimap-hide 0.3.0 
minimap-highlight-selected 4.5.0 
minimap-linter 1.6.0 
minimap-pigments 0.2.2 
minimap-split-diff 0.3.2 
multi-cursor 2.1.5 
multi-cursor-plus 1.2.0 
multi-file-open 0.1.1 
multirow-tabs 0.3.3 
nav-panel 0.0.11 
navigate 0.1.17 
navigate-indent 0.1.0 
next-occurrence 0.5.0 
nuclide undefined 
open-git-modified-files 0.2.4 
open-recent 5.0.0 
open-this 0.4.0 
pdf-view 0.54.0 
php-debug 0.2.4 
php-extract-method 0.2.1 
php-getters-setters 0.5.5 
pigments 0.39.0 
preview 0.17.0 
project-folder 1.2.1 
project-view 0.6.1 
promote-folder-to-project 2.0.1 
punchclock 0.6.5 
python-isort 0.0.7 
python-tools 0.6.9 
qolor 0.4.1 
quantum-shell 0.7.1 
recent-projects 0.13.1 
refactor 0.11.3 
regex-tester 0.2.0 
remember-file-positions 0.2.2 
remote-edit 1.8.24 
revert-buffer 0.5.0 
script 3.14.1 
seti-ui 1.6.0 
show-origin 0.2.0 
sloc 1.1.0 
split-diff 1.1.1 
switch-header-source 0.22.0 
symbols-tree-view 0.13.2 
term 0.2.4 
text-manipulation 0.6.0 
todo-show 1.9.0 
tree-view-git-branch 0.1.2 
tree-view-git-status 1.3.0 
up2date 0.1.2 
vim-mode-clipboard-plus 0.2.0 
vim-mode-plus 0.82.0 
w3c-validation 0.4.0 
xkcd-comics 0.3.0 
xml-tools 0.2.1 
```

**Second error**

1. see above
2. Attempt to close the offending tab

**Atom**: 1.14.0 x64 (**no problems with 1.13.1**)
**Electron**: 1.3.13
**OS**: Debian GNU/Linux
**Thrown From**: [tabs](https://github.com/atom/tabs) package 0.104.1


### Stack Trace

Uncaught TypeError: Cannot read property 'destroy' of undefined

```
At /usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/display-marker-layer.js:189

TypeError: Cannot read property 'destroy' of undefined
    at DisplayMarkerLayer.module.exports.DisplayMarkerLayer.destroyMarker (/app.asar/node_modules/text-buffer/lib/display-marker-layer.js:189:34)
    at /app.asar/node_modules/text-buffer/lib/marker-layer.js:366:37
    at Set.forEach (native)
    at MarkerLayer.module.exports.MarkerLayer.destroyMarker (/app.asar/node_modules/text-buffer/lib/marker-layer.js:365:34)
    at Marker.module.exports.Marker.destroy (/app.asar/node_modules/text-buffer/lib/marker.js:269:18)
    at DisplayMarker.module.exports.DisplayMarker.destroy (/app.asar/node_modules/text-buffer/lib/display-marker.js:27:25)
    at /app.asar/node_modules/text-buffer/lib/display-marker-layer.js:50:23
    at Set.forEach (native)
    at DisplayMarkerLayer.module.exports.DisplayMarkerLayer.clear (/app.asar/node_modules/text-buffer/lib/display-marker-layer.js:49:40)
    at DisplayMarkerLayer.module.exports.DisplayMarkerLayer.destroy (/app.asar/node_modules/text-buffer/lib/display-marker-layer.js:37:12)
    at displayMarkerLayersById.forEach (/app.asar/node_modules/text-buffer/lib/display-layer.js:115:59)
    at Map.forEach (native)
    at DisplayLayer.destroy (/app.asar/node_modules/text-buffer/lib/display-layer.js:115:34)
    at TextEditor.module.exports.TextEditor.destroyed (/app.asar/src/text-editor.js:540:25)
    at TextEditor.module.exports.Model.destroy (/app.asar/src/model.js:31:58)
    at Pane.module.exports.Pane.destroyItem (/app.asar/src/pane.js:660:18)
    at TabBarView.module.exports.TabBarView.closeTab (/app.asar/node_modules/tabs/lib/tab-bar-view.js:333:26)
    at HTMLDivElement.TabView.didClickCloseIcon (/app.asar/node_modules/tabs/lib/tab-bar-view.js:222:19)
```

### Commands

```
     -3:07.6.0 click (a.btn-copy-report.icon.icon-clippy)
     -3:06.0 blur (a.btn-copy-report.icon.icon-clippy)
     -3:05.9.0 click (a.btn-issue.btn.btn-error)
  3x -3:04.7.0 blur (a.btn-issue.btn.btn-error)
     -2:37.2.0 click (div.close.icon.icon-x)
  3x -2:37.2.0 blur (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack.remove)
  4x -2:35.9.0 core:move-left (input.hidden-input)
     -2:34 click (span.syntax--punctuation.syntax--definition.syntax--function.syntax--tex)
  3x -2:31.8.0 blur (input.hidden-input)
  2x -2:03.7.0 click (div.line)
  9x -2:00.2.0 blur (input.hidden-input)
     -1:38 settings-view:open (input.hidden-input)
  9x -1:37.7.0 blur (input.hidden-input)
     -1:01.6.0 click (div.close-icon)
  3x -0:55.6.0 blur (ul.list-inline.tab-bar.inset-panel)
     -0:01.1.0 click (a.btn-copy-report.icon.icon-clippy)
```

### Non-Core Packages

```
atom-beautify 0.29.17 
atom-css-unit-converter 1.1.1 
atom-php-navigation 0.6.0 
atom-ternjs 0.17.0 
atom-x 0.4.0 
atomatigit 1.5.5 
autoclose-html 0.23.0 
autocomplete-awk 0.1.2 
autocomplete-bibtex 0.7.0 
autocomplete-clang 0.10.0 
autocomplete-html-entities 0.1.0 
autocomplete-paths 1.0.5 
autocomplete-php 0.3.7 
autocomplete-python undefined 
autocomplete-xml 0.9.4 
autoupdate-packages 1.3.1 
block-comment 0.4.3 
branch-status 0.8.3 
character-table 0.4.3 
clipboard-plus 0.5.1 
color-picker 2.2.5 
column-select 0.2.0 
compare-files 0.8.0 
css-snippets 1.1.0 
ctrl-last-tab 0.3.0 
docblockr 0.9.1 
emmet 2.4.3 
emmet-snippets-compatibility 1.0.6 
es-navigation 0.1.5 
es6-javascript 1.0.0 
ex-mode 0.13.1 
file-icons 2.0.15 
filesize 2.0.4 
fixmyjs 1.4.0 
fold-comments 0.6.0 
format-sql 0.3.0 
git-blame 0.4.11 
git-clone 0.2.0 
git-control 0.8.2 
git-diff-details 1.4.0 
git-difftool 0.2.7 
git-go 1.0.5 
git-grep 0.9.0 
git-hide 1.0.0 
git-history 3.2.0 
git-log 0.4.1 
git-plus 7.2.1 
git-status 0.3.4 
git-time-machine 1.5.4 
gitignore-snippets 0.2.5 
goto 1.8.3 
goto-definition 1.1.19 
highlight-selected undefined 
hyperclick 0.0.40 
hyperclick-php 0.2.1 
indent-toggle-on-paste 0.1.3 
iso8601-timestamp 0.1.1 
javascript-snippets 1.2.1 
js-hyperclick 1.10.1 
js-refactor undefined 
keyboard-localization 1.5.0 
language-apache 1.7.0 
language-diff 0.7.0 
language-gettext 0.6.1 
language-latex 1.0.0 
language-sql-mysql 0.3.1 
latex-autocomplete 1.0.0 
latex-completions 0.3.2 
latex-friend 0.0.11 
latex-hyperclick 0.2.0 
latexer 0.3.0 
latextools 0.8.5 
lcars-syntax 0.0.1 
less-autocompile 1.0.0 
less-than-slash 0.16.0 
line-count-status 0.0.3 
line-jumper 0.14.1 
linter 1.11.21 
linter-checkbashisms 0.1.5 
linter-chktex 1.3.0 
linter-csslint 1.3.3 
linter-htmlhint 1.3.1 
linter-jshint 3.0.2 
linter-less 2.6.0 
linter-mixed-indent 1.1.0 
linter-mypy 2.0.3 
linter-php 1.3.1 
linter-pylint 1.2.2 
linter-shellcheck 1.4.3 
linter-xmllint 1.4.0 
local-history 4.0.1 
man 0.8.2 
merge-conflicts 1.4.4 
method-separator 0.1.4 
minimap 4.26.8 
minimap-cursorline 0.2.0 
minimap-find-and-replace 4.5.1 
minimap-git-diff 4.3.1 
minimap-hide 0.3.0 
minimap-highlight-selected 4.5.0 
minimap-linter 1.6.0 
minimap-pigments 0.2.2 
minimap-split-diff 0.3.2 
multi-cursor 2.1.5 
multi-cursor-plus 1.2.0 
multi-file-open 0.1.1 
multirow-tabs 0.3.3 
nav-panel 0.0.11 
navigate 0.1.17 
navigate-indent 0.1.0 
next-occurrence 0.5.0 
nuclide undefined 
open-git-modified-files 0.2.4 
open-recent 5.0.0 
open-this 0.4.0 
pdf-view 0.54.0 
php-debug 0.2.4 
php-extract-method 0.2.1 
php-getters-setters 0.5.5 
pigments 0.39.0 
preview 0.17.0 
project-folder 1.2.1 
project-view 0.6.1 
promote-folder-to-project 2.0.1 
punchclock 0.6.5 
python-isort 0.0.7 
python-tools 0.6.9 
qolor 0.4.1 
quantum-shell 0.7.1 
recent-projects 0.13.1 
refactor 0.11.3 
regex-tester 0.2.0 
remember-file-positions 0.2.2 
remote-edit 1.8.24 
revert-buffer 0.5.0 
script 3.14.1 
seti-ui 1.6.0 
show-origin 0.2.0 
sloc 1.1.0 
split-diff 1.1.1 
switch-header-source 0.22.0 
symbols-tree-view 0.13.2 
term 0.2.4 
text-manipulation 0.6.0 
todo-show 1.9.0 
tree-view-git-branch 0.1.2 
tree-view-git-status 1.3.0 
up2date 0.1.2 
vim-mode-clipboard-plus 0.2.0 
vim-mode-plus 0.82.0 
w3c-validation 0.4.0 
xkcd-comics 0.3.0 
xml-tools 0.2.1 
```

**Comment by clearmisp** [comment]
+1, same here with Atom 1.14.0.

**Comment by nathansobo** [comment]
Investigating this. Please fall back to 1.13.x and disable updates in the meantime.

**Comment by nathansobo** [comment]
If people could post their own stack traces and list of non-core packages on this it would be helpful.

**Comment by nathansobo** [comment]
This is fixed in [1.14.1](https://github.com/atom/atom/releases/tag/v1.14.1), which I just published.

**Comment by clare-aire** [comment]
Thank you for the prompt fix!
